core/comm/alias.go

```diff
@@ -0,0 +1,109 @@
+	pkgcomm "github.com/hyperledger/fabric/internal/pkg/comm"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+// functions
+func NewCredentialSupport(rootCAs ...[]byte) *CredentialSupport {
+	return pkgcomm.NewCredentialSupport(rootCAs...)
+}
+
+func NewGRPCClient(config ClientConfig) (*GRPCClient, error) {
+	return pkgcomm.NewGRPCClient(config)
+}
+
+func NewGRPCServer(address string, serverConfig ServerConfig) (*GRPCServer, error) {
+	return pkgcomm.NewGRPCServer(address, serverConfig)
+}
+
+func NewGRPCServerFromListener(listener net.Listener, serverConfig ServerConfig) (*GRPCServer, error) {
+	return NewGRPCServerFromListener(listener, serverConfig)
+}
+
+func NewServerStatsHandler(p metrics.Provider) *ServerStatsHandler {
+	return pkgcomm.NewServerStatsHandler(p)
+}
+
+func AddPemToCertPool(pemCerts []byte, pool *x509.CertPool) error {
+	return pkgcomm.AddPemToCertPool(pemCerts, pool)
+}
+
+func ClientKeepaliveOptions(ka KeepaliveOptions) []grpc.DialOption {
+	return pkgcomm.ClientKeepaliveOptions(ka)
+}
+
+func ExtractCertificateHashFromContext(ctx context.Context) []byte {
+	return pkgcomm.ExtractCertificateHashFromContext(ctx)
+}
+
+func ExtractCertificateFromContext(ctx context.Context) *x509.Certificate {
+	return pkgcomm.ExtractCertificateFromContext(ctx)
+}
+
+func ExtractRawCertificateFromContext(ctx context.Context) []byte {
+	return pkgcomm.ExtractRawCertificateFromContext(ctx)
+}
+
+func GetLocalIP() (string, error) {
+	return pkgcomm.GetLocalIP()
+}
+
+func NewServerTransportCredentials(
+	serverConfig *tls.Config,
+	logger *flogging.FabricLogger) credentials.TransportCredentials {
+
+	return pkgcomm.NewServerTransportCredentials(serverConfig, logger)
+}
+
+func ServerKeepaliveOptions(ka KeepaliveOptions) []grpc.ServerOption {
+	return pkgcomm.ServerKeepaliveOptions(ka)
+}
+
+// vars
+var DefaultConnectionTimeout = pkgcomm.DefaultConnectionTimeout
+var DefaultKeepaliveOptions = pkgcomm.DefaultKeepaliveOptions
+var DefaultTLSCipherSuites = pkgcomm.DefaultTLSCipherSuites
+var ErrClientHandshakeNotImplemented = pkgcomm.ErrClientHandshakeNotImplemented
+var ErrServerHandshakeNotImplemented = pkgcomm.ErrServerHandshakeNotImplemented
+var ErrOverrideHostnameNotSupported = pkgcomm.ErrOverrideHostnameNotSupported
+var MaxRecvMsgSize = pkgcomm.MaxRecvMsgSize
+var MaxSendMsgSize = pkgcomm.MaxSendMsgSize
+
+// types
+type BindingInspector = pkgcomm.BindingInspector
+type CertHashExtractor = pkgcomm.CertHashExtractor
+type ClientConfig = pkgcomm.ClientConfig
+type CredentialSupport = pkgcomm.CredentialSupport
+type DynamicClientCredentials = pkgcomm.DynamicClientCredentials
+type GRPCClient = pkgcomm.GRPCClient
+type GRPCServer = pkgcomm.GRPCServer
+type KeepaliveOptions = pkgcomm.KeepaliveOptions
+type Metrics = pkgcomm.Metrics
+type SecureOptions = pkgcomm.SecureOptions
+type ServerConfig = pkgcomm.ServerConfig
+type ServerStatsHandler = pkgcomm.ServerStatsHandler
+type TLSOption = pkgcomm.TLSOption
```

Comment:
I assume there will be follow-ups to remove these? If not, the move doesn't really do much.

Reply:
> I assume there will be follow-ups to remove these? If not, the move doesn't really do much.

Yep.  Just did not want to change all the consumers at once.

Reply:
> Yep. Just did not want to change all the consumers at once.

Then my suggestion is to hold this out until the next piece is ready. Additional commits should be added until the type aliases are removed.